docs/docsite/rst/dev_guidelines.rst

```diff
@@ -481,19 +481,128 @@ should return in the module.  As well as information related to the call itself,
 some response metadata.  It is OK to return this to the user as well as they may find it useful.
 
 Boto3 returns all values CamelCased.  Ansible follows Python standards for variable names and uses
```

Comment:
```suggestion
Boto3 commonly returns all keys CamelCased.  Ansible follows Python standards for variable names and uses
```

With the growing range of APIs, they're steadily becoming inconsistent, some APIs return camelCase, some return PascalCase.